common/progress.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {ProgressType} from 'vscode-jsonrpc';
+
+export type NgccProgress = {
+  done: false; configFilePath: string; message: string;
+}|{
+  done: true;
+  configFilePath: string;
+};
```

Comment:
@kyliau - `type` declarations should use commas instead of semicolons. I think this might fix the clang-format weirdness.
go/tsstyle#member-property-declarations